src/platform/windows/display_base.cpp

```diff
@@ -156,6 +174,11 @@ namespace platf::dxgi {
           BOOST_LOG(error) << "Unrecognized capture status ["sv << (int) status << ']';
           return status;
       }
+
+      if (config::video.unpaced && mouse_pointer_visible) {
+        // Wait for vblank and flush Desktop Duplication API mouse pointer updates.
+        DwmFlush();
```

Comment:
The "use_dwmflush" has some logic behind it and using DwmFlush bypasses it completely.

Reply:
Yes, and this was intentional.
`DwmFlush()` basically does `WaitForVBlank()` from direct3d and ensures duplication api has the most recent mouse pointer position (for some reason it normally gets delayed). Because this is unpaced path that disables sleeps, we need to limit update rate and call `AcquireNextFrame()` at even intervals for smooth mouse movement,  `WaitForVBlank()` (or in this case `DwmFlush()` accomplishes both.

Reply:
`WaitForVBlank()` and `DwmFlush()` don't behave similarly in terms of resolving capture issues related to the mouse pointer.

`DwmFlush()` was explicitly added to alleviate this issue: https://github.com/loki-47-6F-64/sunshine/issues/227, but using it causes problems if the host monitor refresh rate is lower than the client requested framerate. See the description here: https://github.com/LizardByte/Sunshine/commit/2d969c2cccc61a3fdc718b7aaa0b13410cd850f0

Bypassing the host refresh rate detection before using `DwmFlush()` will cause this issue to reoccur on those configurations. I have not checked this PR, but I would recommend investigating ways to smoothen frametime consistency without involving `DwmFlush()` at all. IMO, it should be removed entirely, as the original mouse-induced stuttering bug can be resolved by invoking `ReleaseFrame()` immediately after acquiring the frame and copying the buffer (see PR on original repo for referene: https://github.com/loki-47-6F-64/sunshine/pull/286)

Reply:
I will see if calling `ReleaseFrame()` before sleeping with `WaitForVBlank()` alleviates mouse delay in the next `AcquireNextFrame()`. If it does, we can safely move from `DwmFlush()` to `WaitForVBlank()` in this particular path.

Reply:
Hard to test this objectively, but looking at mouse trails over recursive moonlight window, this seems to work. Thanks!

Reply:
https://github.com/LizardByte/Sunshine/pull/1203/commits/5e2c20443ba539c975b0cd42440a1bc867436705